### PR TITLE
Remove tomcat webtop logs older than 2 days and other logrotate clean ups

### PR DESCRIPTION
**Pull request by Amygos**
NethServer/dev#5516

### CODE REVIEW

File: root/etc/logrotate.d/webtop [hunk 1]
```diff
@@ -1,12 +1,20 @@
-/var/log/webtop/webtop.log
+/var/log/webtop/webtop.log {
+    copytruncate
+    weekly
+    rotate 52
```
Comment (gsanchietti) [inline root/etc/logrotate.d/webtop]: Remove the rotate from here so the configuration can be inherited from `logrotate.conf`which is controlled by `logrotate{Times}` prop.

Comment (Amygos) [reply root/etc/logrotate.d/webtop]: Ok, do I have to remove the `rotate` prop also from `/etc/logrotate.d/webtop-dav`?

Comment (gsanchietti) [reply root/etc/logrotate.d/webtop]: Yes, thanks

Comment (Amygos) [reply root/etc/logrotate.d/webtop]: :+1:

**Pull request merged**